demisto_sdk/commands/validate/tests/tools_test.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+from demisto_sdk.commands.validate.validators.tools import (
+    collect_all_inputs_from_inputs_section,
+    collect_all_inputs_in_use,
+)
+
+
+@pytest.mark.parametrize(
+    "input_name, expected_result",
+    [
+        (
+            {"0": {"inputs.hello": "test"}, "1": {"inputs.example": "test"}},
+            {"hello: test", "example: test"},
+        ),
+        ({"0": {"inputs": "test"}, "1": {"inputs": "test"}}, set()),
+    ],
+)
+def test_collect_all_inputs_in_use(input_name, expected_result):
+    """
+    Given:
+        - A playbook with inputs in some tasks
+    When:
+        - Running collect_all_inputs_in_use
+    Then:
+        - Return a set of input names and values from any task in the playbook, if the inputs match the pattern inputs.<input_name>
+        scenario 1: inputs.hello: test
+        scenario 2: inputs: test
```

Comment:
scenarios should be written as cases under the given and the then sections. see other tests as an example.

Reply:
Fixed.